package.json

```diff
@@ -65,6 +64,7 @@
   },
   "dependencies": {
     "@types/jest": "^27.0.2",
+    "createjs-module": "^0.8.3",
```

Comment:
Similar to @adagar's comment, I wonder if we needed to move this or if we could keep it in `devDependencies` (if I remember correctly, we were primarily using it for testing purposes).

Reply:
@mmoss @adagar 
`2:1  error  'createjs-module' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies`
I get this linting error if `'createjs-module` is kept in devDependencies.
I guess its because, we use createjs for creating the event to propagate